javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,17 @@
+// Goodboy-Oldboy (A dog age calculator)
+
+/* The same friend (who by the way loves dogs) asks how old his dog will be in 2045. 
+Hmm you think, lets make this into a product as well! 
+Dogs age can either be measured in dog years or in human years, this we want to take into consideration! */
+
+const dogYearOfBirth = 2021;
+const dogYearFuture = 2042;
+const dogYear = Math.round(16 * Math.log(dogYearFuture - dogYearOfBirth) + 31);
```

Comment:
A simple multiplication is sufficient here. `dogYears = 7 * humanYears`. I don't think you need logarithmic operations